pkg/prompt/prompt.go

```diff
@@ -0,0 +1,80 @@
+package prompt
```

Comment:
I find it strange to have a package for `prompt` but not for `table`.

Reply:
cf. #1226 , if we make our own package for table / json it should probably be outside of `cli`